src/system/controlsystem.jl

```diff
@@ -236,7 +236,13 @@ function buildAffineApproximation(f, x, u, w, x̄, ū, w̄, X, U, W)
     L = _getLipschitzConstants(Jxi, xi, sub_rules_Xi)
 
     sub_rules_x̄i = Dict(xi[i] => x̄i[i] for i in 1:(n + m + p))
-    evalSym(x) = Float64.(Symbolics.value.(Symbolics.substitute(x, sub_rules_x̄i)))
+    function evalSym(x)
+        if x isa Vector{SymbolicUtils.BasicSymbolic{Real}}
```

Comment:
What's happening here ? Some comments would be helpful :)
Maybe it's easier to do something like
```julia
y = Symbolics.substitute(x, sub_rules_x̄i)
if eltype(y) <: Real
    return y
end
return  Float64.(Symbolics.value.(y))
```
because `SymbolicUtils.BasicSymbolic{Real}` looks like an internal assumption that might easily break

Reply:
`Float64.(a)` where `a` is a `Vector{BasicSymbolic{Real}}` raises an error
It happens when we execute `test_lazy_ellispoids_abstraction.jl`, in `optimize!`

It's maybe linked to a new version of Symbolics

Reply:
It happens only with `f`, not the jacobians

Reply:
Would the code snippet of my comment work ?

Reply:
No it doesn't work
We really need to subsitute every element of the vector

Reply:
Indeed, I misread, it's fine then if a comment is added to explain